src/prisma/schema.prisma

```diff
@@ -23,6 +23,9 @@ model Configuration {
   maxFeePerGasForRetries             String  @map("maxFeePerGasForRetries")
   maxPriorityFeePerGasForRetries     String  @map("maxPriorityFeePerGasForRetries")
   maxRetriesPerTx                    Int     @map("maxRetriesPerTx")
+  // Contract Indexer Updates
+  indexerListenerCronSchedule        String? @map("indexerListenerCronSchedule")
+  maxBlocksToIndex                   Int     @map("maxBlocksToIndex")
```

Comment:
`maxBlocksToIndex` :   will need a DEFAULT value if you mark it as NOT NULL.

Either mark it as nullable and on code level provide the value which you currently do (i.e., 5 i believe) or add a default value.

otherwise, get this error, when running the server :

```
Database error:
ERROR: column "maxBlocksToIndex" of relation "configuration" contains null values

DbError { severity: "ERROR", parsed_severity: Some(Error), code: SqlState(E23502), message: "column \"maxBlocksToIndex\" of relation \"configuration\" contains null values", detail: None, hint: None, position: None, where_: None, schema: Some("public"), table: Some("configuration"), column: Some("maxBlocksToIndex"), datatype: None, constraint: None, file: Some("tablecmds.c"), line: Some(6046), routine: Some("ATRewriteTable") }
```